app/referee/tasks.py

```diff
@@ -1,74 +1,152 @@
 import json
 import logging
 import math
+from collections import namedtuple
 
 import dask
 import dask.distributed as dd
-
 import ever_given.wrapper
 
 from core import models
 
-
 logger = logging.getLogger(__name__)
 
 
+AnswerPredictionPair = namedtuple("AnswerPredictionPair", ["answer", "prediction"])
```

Comment:
I like this! Saves us from having to do more work than required when things are simple.